### PR TITLE
tests: tests required for alpha

### PR DESCRIPTION
**Pull request by georgiyekkert**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-compute/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Comment by georgiyekkert** [comment]
@alexander-fenster please take a look

File: system-test/compute.js [hunk 1]
```diff
@@ -211,6 +211,46 @@ describe('Compute', () => {
       });
       assert.strictEqual(fetched.shieldedInstanceConfig.enableSecureBoot, true);
     });
+
+    it('API errors propagated', async () => {
```
Comment (bcoe) [inline system-test/compute.js:215]: nit: I like for the assertions to read like a phrase.

`it('propagates API errors')`.

File: system-test/compute.js [hunk 2]
```diff
@@ -211,6 +211,46 @@ describe('Compute', () => {
...
+      });
+      assert.strictEqual(instance.description, 'test');
+      instance.description = '';
+      const [updateOp] = await client.update({
+        project: project,
+        zone: zone,
+        instance: INSTANCE_NAME,
+        instanceResource: instance,
+      });
+      await waitZonalOperation(updateOp);
```
Comment (bcoe) [inline system-test/compute.js:246]: I like the use of `async`/`await` in tests 👍

**Review by bcoe — APPROVED** [review]
LGTM: with @alexander-fenster's approval.

**Review by bcoe — APPROVED** [review]
I should say, LGTM, with @summer-ji-eng or @alexander-fenster's approval.

**Pull request merged**